### ISSUE TITLE
Does't not work in Windows

### ISSUE THREAD

**Issue by AndreySurzhan** [browsertron/pytest-parallel#5]
**Env**: Windows 7.  Python 3.6. 32 bit (Intel)] 

1. Installed `pytest-parallel-0.0.2`
2. run pytest tests with `--worker` argument

**Actual result:** Tests are being run not in parallel 
**Expected result:** several threads should be dedicated for pytest tests.

**Comment by AndreySurzhan** [comment]
@kevlened thanks for quick response. 
`--tests-per-worker` wont work for me since we cannot run different test functions separately from one test `*.py` file due to them depend on each other.

That being said it would be nice to have possibility to define the scope of the run like you can do it in `pytest-xdist` with `--dist` param

**Comment by kevlened** [comment]
Ah, gotcha. In that case, `pytest-xdist` seems like the only option.

As for the `--dist` param, that's  out-of-scope for this package. Distributing across multiple machines is what `pytest-xdist` is good at. `pytest-parallel` is trying to squeeze all possible performance out of a single machine. If you need to run tests in multiple Python versions on a single machine with `pytest-parallel`, [run pytest with `tox`](https://tox.readthedocs.io/en/latest/example/pytest.html).

I'll keep this issue open as an aspirational feature for multiprocessing in Windows.

**Comment by AndreySurzhan** [comment]
Sounds great. Thanks for your work!
